solc/CommandLineParser.h

```diff
@@ -137,7 +139,7 @@ struct CommandLineOptions
 
 	struct
 	{
-		bool prettyJson = false;
+		solidity::util::JsonPrintingFormat prettyJson { JsonPrintingFormat::Format::Compact };
```

Comment:
Since `formatting` is already in the name of the struct, I'd call it just `json`. Also, no point in repeating the default here.
```suggestion
		util::JsonPrintingFormat json;
```